tests/streams/sstream.cpp

```diff
@@ -119,3 +119,32 @@ void strStream::Output_Check()
 
 // Register the stream sub suite, by using some stream helper macro.
 STREAM_TEST_SUBSUITE_NAMED_REGISTRATION(strStream)
+
+TEST_CASE("wxStringOutputStream::Tell", "[stream]")
+{
+    wxStringOutputStream ss;
+    CHECK( ss.TellO() == 0 );
+
+    const char* const s = "Hello world";
+
+    ss.Write(s, strlen(s));
+    const wxFileOffset pos = ss.TellO();
+    CHECK( pos != 0 );
```

Comment:
`pos != 0` is a pretty low bar... I get that you're using the result in the next test, but shouldn't the result really be `strlen(s)`?

Reply:
Yes, thanks, I initially wanted to reuse the same code for both the UTF-8 and UTF-16 checks and then the position would differ, but with the way it ended up being written, it should indeed be just checked directly, I'll update this.